conf/deployment/vsphere/ipi_1az_rhcos_lowerreq_3m_3w.yaml

```diff
@@ -0,0 +1,16 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
```

Comment:
I'm not sure as thats what we have it set to within the UPI counterpart [config](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/vsphere/upi_1az_rhcos_lowerreq_vmfs_3m_3w.yaml)

Reply:
@petr-balogh is setting this config value to true necessary for lowerreq configs? If so I think the UPI counterpart might be misconfigured.

Reply:
@jimdevops19 I remember that this requires UI deployment to be used and ODF does the trick itself. There is no way to mimic this behavior from CLI deployment the same way how ODF does that. @jimdevops19 might remember more about this.  If this is the case, not sure if we should contain the config value to enable UI deployment directly in this production configuration file. Our jenkins job has this option enabled by default for lowerreq jobs iirc.

Reply:
I think it's worth to add here the ui_deployment conf value like here:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/ui_deployment.yaml

Because this feature cannot be tested otherwise as we discussed offline.